examples/components/shared/aml_comm.py

```diff
@@ -0,0 +1,722 @@
+##########################################################################################
```

Comment:
Let’s add an e2e test in the CI/CD that triggers at least [one example pipeline](https://github.com/Azure-Samples/azure-ml-federated-learning/blob/main/.github/workflows/pipeline-e2e-test.yaml#L457) using the shared folder components when any change is detected in the shared folder.

Reply:
@garg-amit what's the reasoning behind? The files from shared forlder are never used as of now and whether it is run in CI/CD pipeline in any of the examples is purely dependent on the default config file. Therefore, if any of the files changes the corresponding e2e pipeline is triggered.